Libraries/Map/Map.h

```diff
@@ -36,13 +34,13 @@ class Map {
     uint8_t currentColumn();
     // Returns a uint8_t representing the orientation the robot is looking at (1=North, 2=East, 3=South, 4=West).
     uint8_t getOrientation();
-    // Returns a uint8_t with number of tiles that are accessible and the robot has not visited.
-    uint8_t getUnvisitedTiles();
+    // Recieves an array of maps and returns a uint8_t with number of tiles that are accessible and the robot has not visited.
+    uint8_t getUnvisitedTiles(TVector<Map> tiles_map);
```

Comment:
Maybe a good later optimization would be to have a member variable counting the unvisitedtiles, instead of having to check all the map always. You can add this as a TODO.

Reply:
Ok.